### PR TITLE
🌐 [i18n-KO] Translated `perf_hardware.md` to Korean

### PR DESCRIPTION
**Pull request by augustinLib**
# What does this PR do?

Translated the `perf_hardware.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)
@0525hhgus, @Sunmin0520, @54data, @seank021, @kihoon71

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

## Who can review? (Final)

May you please review this PR? @sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
빠른 번역 감사합니다 👍
아래 목차와 누락된 문장 관련하여 우선적으로 리뷰를 올렸으니, 참고 부탁드립니다!
(52 라인까지 검토)

File: docs/source/ko/perf_hardware.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+모델 훈련과 추론에 사용하는 하드웨어는 성능에 큰 영향을 미칠 수 있습니다. GPU에 대해 자세히 알아보려면, Tim Dettmer의 훌륭한 블로그 포스트를 확인해보세요. [블로그 포스트 링크](https://timdettmers.com/2020/09/07/which-gpu-for-deep-learning/) (영어로 작성됨).
+
+GPU 설정에 대한 실용적인 조언을 살펴보겠습니다.
+
+## GPU [[gpu]]
+더 큰 모델을 훈련시킬 때는 기본적으로 세 가지 옵션이 있습니다:
+- 더 큰 GPU
+- 더 많은 GPU
+- 더 많은 CPU 및 NVMe (DeepSpeed-Infinity를 통한 offload)
```
[the remark this answers is not in the document]
Comment (augustinLib) [reply docs/source/ko/perf_hardware.md:32]: 이 부분은 생각을 못했었네요 ㅠㅠ 다음부터는 이러한 부분도 신경써서 올리겠습니다! ㅠㅠ

**Comment by Sunmin0520** [comment]
위에서 @0525hhgus 님의 댓글을 제외하고는 별도의 리뷰사항 없습니다 :)

File: docs/source/ko/perf_hardware.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+또한 GPU가 안정적인 전압을 받을 수 있도록 고급 PSU를 선택해야 합니다. 저품질의 PSU는 GPU가 최고 성능으로 동작하기 위해 필요한 전압을 안정적으로 공급하지 못할 수 있습니다.
+
+물론, PSU는 GPU를 구동시키기에 충분한 여분의 전력 용량을 가져야 합니다.
+
+**냉각**:
+
+GPU가 과열되면 성능이 저하되고 최대 성능을 발휘하지 못할 수 있으며, 너무 뜨거워지면 중지될 수 있습니다.
+
+GPU가 과열될 때 정확한 적정 온도를 알기 어려우나, 아마도 +80도C 미만이면 좋지만 더 낮을수록 좋습니다. 70℃-75℃ 정도가 훌륭한 온도 범위입니다. 성능 저하가 발생하기 시작하는 온도는 대략 84℃-90℃ 정도일 것입니다. 하지만 성능 저하 이외에도 지속적으로 매우 높은 온도는 GPU 수명을 단축시킬 수 있습니다.
```
Comment (54data) [inline docs/source/ko/perf_hardware.md:58]:
```suggestion
GPU가 과열될 때 정확한 적정 온도를 알기 어려우나, 아마도 +80℃ 미만이면 좋지만 더 낮을수록 좋습니다. 70℃-75℃ 정도가 훌륭한 온도 범위입니다. 성능 저하가 발생하기 시작하는 온도는 대략 84℃-90℃ 정도일 것입니다. 하지만 성능 저하 이외에도 지속적으로 매우 높은 온도는 GPU 수명을 단축시킬 수 있습니다.
```
해당 부분도 다른 표기와 같이 맞춰주는 게 좋을 것 같아서 제안 드립니다!

**Review by ArthurZucker — APPROVED** [review]
Looks good! Could you update the ## GPU [[gpu]] in the english version as well? Seems like it is not rendering properly, let's kill two birds with one stone! 

image

**Comment by augustinLib** [comment]
> Looks good! Could you update the ## GPU [[gpu]] in the english version as well? Seems like it is not rendering properly, let's kill two birds with one stone!
> 
> image

@ArthurZucker 
I updated that issue! and also i could confirm [here](https://huggingface.co/docs/transformers/main/en/perf_hardware) that it worked successfully. 
This part didn't work well in previous versions, but it works well in the current `main`. It was a problem that appeared because the newline was omitted in the markdown file

Thank you for your hard work and dedication, it is greatly appreciated.

**Pull request merged**